src/cephadm/cephadm

```diff
@@ -7401,6 +7407,11 @@ def _get_parser():
     parser_pull = subparsers.add_parser(
         'pull', help='pull latest image version')
     parser_pull.set_defaults(func=command_pull)
+    parser_pull.add_argument(
+        '--insecure',
```

Comment:
```suggestion
        '--registry-insecure',
```

... or maybe drop this option from bootstrap and require a pull before bootstrap?